src/main/java/org/team1540/robot2023/commands/arm/ManualArm.java

```diff
@@ -0,0 +1,36 @@
+package org.team1540.robot2023.commands.arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+
+public class ManualArm extends CommandBase {
+    Arm arm;
+    CommandXboxController controller;
+
+    public ManualArm(Arm arm, CommandXboxController controller){
+        this.arm = arm;
+        this.controller = controller;
+        addRequirements(arm);
+    }
+
+    @Override
+    public void initialize() {
+        arm.setManualControl(true);
+    }
+
+    @Override
+    public void execute() {
+        arm.setRotatingSpeed(controller.getLeftY()); // TODO: 2/11/2023 Check angles here
```

Comment:
I would think about squaring or even cubing this to allow for better precision control